upcoming-release-notes/454.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Enhancements
+authors: [DirgoSalga]
+---
+
+Add integration of Rheinhessen Sparkasse (SPK_WORMS_ALZEY_RIED_MALADE51WOR) to the GoCardless app
```

Comment:
I think this needs backticks to prevent the spellcheck from complaining at the next release

```suggestion
Add integration of Rheinhessen Sparkasse (`SPK_WORMS_ALZEY_RIED_MALADE51WOR`) to the GoCardless app
```